templates/include/telemetry/environment.1.schema.json

```diff
@@ -357,6 +357,29 @@
         "e10sCohort": {
           "type": "string"
         },
+        "intl": {
+          "type": "object",
+          "properties": {
+            "requestedLocales": {
+              "type": "string"
```

Comment:
I think these should be:

```json
"requestedLocales": {
  "type": "array",
  "items": {
    "type": "string"
  }
```

Reply:
Ah, I missed we have an array type. I think you're right

Reply:
I force pushed since GH these days shows them anyway in the history